pinot-tools/src/main/java/org/apache/pinot/tools/BootstrapTableTool.java

```diff
@@ -31,7 +31,7 @@
 import java.util.Map;
 import org.apache.commons.io.FileUtils;
 import org.apache.pinot.common.minion.MinionClient;
-import org.apache.pinot.core.common.MinionConstants;
+import org.apache.pinot.common.minion.MinionConstants;
```

Comment:
This change will incur backward incompatibility if the library pulls `MinionConstants`. Can you tag the `release-note` + `backward incompatibility` for this PR?

Reply:
As discussed offline, removed the validation check and this pr will be backward compatible. Will file another pr for the validation.